applications.json

```diff
@@ -7,9 +7,9 @@
       ],
       "repo_url" : "https://github.com/friedrichweise/wechsel",
       "title" : "wechsel",
-      "icon_url" : "https://github.com/friedrichweise/wechsel/blob/master/wechsel/Assets.xcassets/AppIcon.appiconset/Artboard-2.png ",
+      "icon_url" : "https://github.com/friedrichweise/wechsel/blob/master/wechsel/Assets.xcassets/AppIcon.appiconset/Artboard-2.png",
```

Comment:
@serhii-londar I'm not sure, but shouldn't this one be direct to the image?

Reply:
I've changed it